src/containers/ProjectContentsContainer.jsx

```diff
@@ -53,6 +59,9 @@ class ProjectContentsContainer extends Component {
           <ModalBody>
             <p>{this.state.translationText}</p>
             <input autoFocus placeholder="Translate some text" />
+            <DefaultButton onClick={this.handleSubmit}>
+              Submit
+            </DefaultButton>
           </ModalBody>
           <ModalFooter>
             <DefaultButton onClick={() => this.setState({modalOpen: false})}>
```

Comment:
I'd recommend committing lock file changes separately from other changes. Otherwise, you'll find yourself in a world of pain if you want to revert just the lock file changes, or even delete them from the commit history of a branch.